### PR TITLE
Add missing documentation for "pairing" command

### PR DESCRIPTION
**Pull request by kuc**

### CODE REVIEW

File: src/chesster.js [hunk 1]
```diff
@@ -133,6 +133,7 @@ function prepareCommandsMessage(){
 									  users.getIdString("chesster") + ": \n```" +
         "    [ starter guide ]              ! get the starter guide link; thanks GnarlyGoat!\n" +
         "    [ rules | regulations ]        ! get the rules and regulations.\n" + 
+        "    [ pairing | pairing <player> ] ! get your (or given <player>) latest pairings with scheduled time" +
```
Comment (lakinwecker) [inline src/chesster.js]: Needs a '\n' at the end of the string.

**Comment by endrawes0** [comment]
Fixed #135, right?

**Comment by endrawes0** [comment]
Who is this btw? Just keeping track of contributors.

Thanks!

File: src/chesster.js (hunk 1 above)
Comment (kuc) [inline src/chesster.js]: My bad. I will fix this.

**Comment by kuc** [comment]
@endrawes0 Believe me or not, but I was planned to fix this from 2 weeks, and I haven't even noticed that issue #135 :smile:
What a coincidence.

**Comment by lakinwecker** [comment]
Looks good to me. :)

**Comment by lakinwecker** [comment]
We need to up the timeouts, there is still one test that will _sometimes_ take to long and cause the travis build to fail.

I have restarted the build for now.

**Comment by endrawes0** [comment]
Seems to happen about 40% of the time

**Comment by endrawes0** [comment]
Thanks mystery contributor!

**Comment by kuc** [comment]
No problem :wink:

**Pull request merged**